src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1203,16 +1217,17 @@ void set_signal_handler(int sig) {
   struct sigaction oldAct;
   sigaction(sig, (struct sigaction*)NULL, &oldAct);
 
+  // Query the current signal handler. Needs to be a separate operation
+  // from installing a new handler since we need to honor AllowUserSignalHandlers.
   void* oldhand = get_signal_handler(&oldAct);
```

Comment:
It's a pre-existing issue, but I dislike the **"old"** in **oldhand** and **oldAct**. It makes it sound like it is a cached or previous value, which is especially confusing when in the comment we refer to it as the **current** one. Any chance we can clean this up in this fix?

Reply:
I think it stems from the Posix interface calling the parameters `act` and `oact` and ppl are used to that naming:
```
int sigaction(int sig, const struct sigaction *restrict act,
       struct sigaction *restrict oact);
```
That said, we use a bunch of different names, like oact, oldAct or oldSigAct, which I disliked. I wanted to unify those namings but refrained since I wanted to keep the patch small. Its difficult enough as it is.

Since this would be an easy change in itself, can we keep it for a different RFE? As for name, lets say currentAct?

Reply:
I'd love if it we could follow up on it.